s2-sdk/src/main/java/s2/config/Config.java

```diff
@@ -22,7 +23,8 @@ private Config(
       Integer maxRetries,
       Duration requestTimeout,
       Duration retryDelay,
-      String userAgent) {
+      String userAgent,
+      Boolean compression) {
```

Comment:
nitpick, but could you keep this sequence of config options alphabetized (here and elsewhere in this config class and builder)?

Reply:
Updated.